tasks/_inc_client_build_steps.yml

```diff
@@ -0,0 +1,32 @@
+---
+
+- name: Install packages with yarn
+  yarn:
+    executable: "yarn --network-timeout 300000 --check-files"
+    path: "{{ galaxy_server_dir }}/client"
+  environment:
+    PATH: "{{ galaxy_venv_dir }}/bin:{{ ansible_env.PATH }}"
+    VIRTUAL_ENV: "{{ galaxy_venv_dir }}"
+
+- name: Run gulp
+  command: gulp
```

Comment:
This relies on the gulp cli, right?  I'd probably use the package scripts directly and rely on `yarn run gulp` here.